client/src/pages/spectrogram/components/annotation/__tests__/Actions.test.tsx

```diff
@@ -196,8 +199,8 @@ describe('Annotation list component', () => {
     await userEvent.click(updateButton);
 
     // Assert
-    const modal = await screen.findByLabelText('Annotation 0 Modal');
-    expect(modal).toHaveClass('modal-open');
+    const modal = await screen.queryByLabelText('Modal Dialog');
+    expect(modal).toBeInTheDocument();
```

Comment:
does toBeVisible() pass?

Reply:
Good point actually, this shouldn't be visible in this test as the update should close it. Will remove!